client/model/note.go

```diff
@@ -125,6 +126,14 @@ func addNoteOrRest(score *Score, noteOrRest ScoreUpdate) error {
 					part.Octave, part.KeySignature, part.Transposition,
 				)
 
+				if midiNote < 0 || midiNote > 127 {
+					log.Debug().
+						Int("midiNote", int(midiNote)).
+						Msg("Midi note out of range.")
```

Comment:
This debug log has the same information as the user-facing error message, so I think we can remove it to reduce noise.

Reply:
Thanks for the feedback, I've checked the Slack. Working on it!